### DISCUSSION TITLE
Does VPNHotspot work fine alongside AFWall+?

### DISCUSSION THREAD

**Discussion by poqdavid** [Mygod/VPNHotspot#536] [Q&A] [answered]
Hi,

I want to use AFWall+ with VPNHotspot and I am not 100% sure if it breaks VPNHotspot or not.
does anyone know anything about this?
and is there a specific configuration I need to do on AFWall+ to make sure there won't be a problem?

**Comment by Mygod** [comment, marked as answer]
According to #73 it works.
